stubs/Nette/Application/UI/Multiplier.stub

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
Not sure why would we need this if we would have collector

Reply:
Collector see return type of createComponent or expression in return - which is not always expression in which Multiplier is created. I see no other way that having generics for that.